app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
```

Comment:
- Each import value must be on a line by itself and open/close parenthesis must be on separate line. For example
```
from app.errors import (
    NotWearingMaskError,
    NotVaccinatedError,
    OutdatedVaccineError,
)
```
- Please, add trailing comma after `OutdatedVaccineError`